client/components/sidebar-navigation/index.jsx

```diff
@@ -2,17 +2,23 @@
  * External dependencies
  */
 import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
 
 /**
  * Internal Dependencies
  */
-import layoutFocus from 'lib/layout-focus';
 import Gridicon from 'components/gridicon';
+import { setLayoutFocus } from 'state/ui/layout-focus/actions';
+
+class SidebarNavigationSection extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.toggleSidebar = this.toggleSidebar.bind( this );
```

Comment:
Was this broken previously? I could see how it might have been. Just curious about the changes here, and to the name of the component.